app/src/sharedTest/java/org/oppia/app/topic/TopicFragmentTest.kt

```diff
@@ -53,7 +53,7 @@ class TopicFragmentTest {
   @Test
   fun testTopicFragment_toolbarTitle_isDisplayedSuccessfully() {
     launchTopicActivityIntent(internalProfileId, FRACTIONS_TOPIC_ID).use {
-      onView(allOf(instanceOf(TextView::class.java), withParent(withId(R.id.topic_toolbar))))
```

Comment:
Create a new file `TopicActivityTest` and all test cases which are related to `TopicActivity` and not `TopicFragment` shift them to that file. 

Also, add test case to check that the click on `NavigationDrawer` is working correctly.

Basically we want drawer and toolbar test cases in `TopicActivityTest` and tabs and child fragment test cases in `TopicFragmentTest` similar to how UI has been laid out.